src/main/java/appeng/client/gui/widgets/MEGuiTextField.java

```diff
@@ -43,6 +43,7 @@
 	private final int _yPos;
 	private final int _width;
 	private final int _height;
+	private final int _fontPad;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_fontPad" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00116)